DSA.html

```diff
@@ -10,10 +10,15 @@
         }
 
         .button1 {
-            color: black;
-            font-size: medium;
-            border-radius: 8px;
-
+            color: #fff;
```

Comment:
As per our discussion in #213 you only need to change the hyperlink so don't change these styles.